fastMCKalman/MC/fastSimulation.cxx

```diff
@@ -215,7 +294,7 @@ Bool_t AliExternalTrackParam4D::GetXYZatR(Double_t xr,Double_t bz, Double_t *xyz
 }
 /// get radiial direction sign
 Int_t AliExternalTrackParam4D::GetDirectionSign(){
-  Float_t dir = GetX()*Px()+GetY()*Py();
+  Float_t dir = GetX()*Px()+GetY()*Py()+GetZ()*Pz();
```

Comment:
Should be reverted- this function should check of the moneta is out or in in cylindrical coordinate

Reply:
Got the function back to how it was